src/suites/cts/command_buffer/copies.spec.ts

```diff
@@ -109,3 +109,67 @@ g.test('b2t2t2b', async t => {
 
   t.expectContents(dst, data);
 });
+
+g.test('imageBitmapToTexture', async t => {
+  const img = document.createElement('img');
+  img.src = '../../assets/img/test.png';
+  await img.decode();
+
+  const imageBitmap = await createImageBitmap(img, 0, 0, img.width, img.height);
+  const dst = t.device.createTexture({
+    size: {
+      width: img.width,
+      height: img.height,
+      depth: 1,
+    },
+    format: 'rgba8uint',
+    usage: GPUTextureUsage.COPY_DST | GPUTextureUsage.COPY_SRC,
+  });
+
+  // TODO(shaobo.yan@intel.com) : remove as any after @webgpu/types updating
+  (t.device.defaultQueue as any).copyImageBitmapToTexture(
```

Comment:
This is available now, can you update `package.json` to point to `@webgpu/types` 0.0.19?

Reply:
Sure. Previously I think the roll version patch should be a owner's right :). I'll do it.

Reply:
Done.